docs/getting-started-publishing.md

```diff
@@ -306,3 +306,43 @@ GitHub enterprise installations should work in the same manner as github.com; yo
 | `GITHUB_HOST` | The hostname for the GitHub enterprise server. |
 
 Alter your `siteConfig.js` to add a property `'githubHost'` which represents the GitHub Enterprise hostname. Alternatively, set an environment variable `GITHUB_HOST` when executing the publish command.
+
+## Deploying with Surge
+
+Surge is a [static web hosting platform](https://surge.sh/help/getting-started-with-surge) , it is used to deploy your Docusaurus project from command line in a minute. Deploying your project to surge is easy and it’s also free (including a custom domain and SSL).
+
+Deploy your app in a matter of seconds using surge with the following steps:
+
+1. First, install surge using npm by running the following command
+
+```bash
+npm install --global surge
+```
+
+2. Run a single command inside the root directory of your project
```

Comment:
Does this require the site to be built first? Aka does the `build` directory have to be populated with the generated files?

Reply:
Yes